xchange-bx/src/main/java/org/knowm/xchange/bx/service/BxAccountService.java

```diff
@@ -0,0 +1,52 @@
+package org.knowm.xchange.bx.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.bx.BxAdapters;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.FundingRecord;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+import org.knowm.xchange.service.trade.params.WithdrawFundsParams;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+
+public class BxAccountService extends BxAccountServiceRaw implements AccountService {
+
+    public BxAccountService(Exchange exchange) {
+        super(exchange);
+    }
+
+    @Override
+    public AccountInfo getAccountInfo() throws IOException {
+        return new AccountInfo(BxAdapters.adaptWallet(getBxBalance()));
+    }
+
+    @Override
+    public String withdrawFunds(Currency currency, BigDecimal bigDecimal, String s) throws IOException {
+        return null;
```

Comment:
Instead of return null, common practice is to raise `NotYetImplementedForExchangeException` or `NotAvailableFromExchangeException`. The same is for other not implemented methods.